packages/hint-no-protocol-relative-urls/tests/tests.ts

```diff
@@ -76,4 +76,4 @@ const tests: HintTest[] = [
     }
 ];
 
-hintRunner.testHint(getHintPath(__filename), tests);
+hintRunner.testHint(getHintPath(__filename), tests, { parsers: ['html'] });
```

Comment:
Why is this the only hint that needs this?

Reply:
Let me ckeck, but maybe I just missed that one